net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -144,23 +181,41 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		--without-seccomp
```

Comment:
What's this about?

Reply:
Right, upstream removed the seccomp-based sandbox in https://github.com/bitcoin/bitcoin/commit/32e2ffc39374f61bb2435da507f285459985df9e